backend/src/email/routes/tests/email-transactional.routes.test.ts

```diff
@@ -31,7 +52,10 @@ beforeAll(async () => {
   )
 })
 
-afterEach(() => jest.resetAllMocks())
```

Comment:
not very sure why this was chosen, but I had to change to `restoreAllMocks` to get rid of side effects so that tests running in series would pass. (before that, they would pass individually but not as a group.)